src/routes/[...404].tsx

```diff
@@ -0,0 +1,8 @@
+import { useLocation } from "@solidjs/router";
+import { getLocaleFromPathname, missingTranslationMessage } from "~/i18n";
+
+export default function NoEntry() {
+	const { pathname } = useLocation();
+
+	return <p>{missingTranslationMessage(getLocaleFromPathname(pathname))}</p>;
```

Comment:
Also, what do you think of adding a relevant Title component with a relevant message?

Reply:
I figured keeping the title of the entry we want to ask the user to translate is nicer UX